responsibleai/tests/counterfactual/test_counterfactual_advanced_features.py

```diff
@@ -47,19 +45,19 @@ def test_counterfactual_vary_features(
             feature_importance=feature_importance)
         model_analysis.counterfactual.compute()
 
-        cf_obj = model_analysis.counterfactual.get()[0]
-        for feature_name in feature_names:
-            if not vary_all_features and feature_name != feature_names[0]:
-                assert np.all(
-                    cf_obj.cf_examples_list[0].final_cfs_df[feature_name] ==
-                    X_test.iloc[0:1][feature_name][0])
-            else:
-                assert np.any(
-                    cf_obj.cf_examples_list[0].final_cfs_df[feature_name] !=
-                    X_test.iloc[0:1][feature_name][0])
+        # TODO: The logic below needs to be made robust for gated tests
```

Comment:
Could you use `np.isclose` in this PR?
https://numpy.org/doc/stable/reference/generated/numpy.isclose.html

Reply:
After discussion, this can be done in a follow-up

Reply:
Opened issue https://github.com/microsoft/responsible-ai-widgets/issues/758 to track this.